static/js/zamboni/reviewers.js

```diff
@@ -303,19 +303,22 @@ function initExtraReviewActions() {
     }),
   );
 
-  $('#toggle_attachment_file').on('click', function (e) {
-    e.preventDefault();
-    $('#attachment-type-toggle').addClass('hidden');
-    $('#attachment_input_wrapper').addClass('hidden');
+  const showFileWrapper = (e) => {
+    e?.preventDefault();
+    $('#attachment-type-toggle, #attachment_input_wrapper').addClass('hidden');
     $('#attachment_file_wrapper').removeClass('hidden');
-  });
-
-  $('#toggle_attachment_input').on('click', function (e) {
-    e.preventDefault();
-    $('#attachment-type-toggle').addClass('hidden');
-    $('#attachment_file_wrapper').addClass('hidden');
+  };
+  const showInputWrapper = (e) => {
+    e?.preventDefault();
+    $('#attachment-type-toggle, #attachment_file_wrapper').addClass('hidden');
     $('#attachment_input_wrapper').removeClass('hidden');
-  });
+  };
+
+  $('#id_attachment_file').prop('files').length && showFileWrapper();
```

Comment:
My jquery days are long gone but my understanding is this code will execute one time on page load, or when that element is in the dom? It's not totally clear since this is no explicit event trigger. Is it possible to do that somehow or maybe add a comment indicating when this code is/should be executed?

Reply:
Once on page load, yeah -- `initExtraReviewActions()` is called on document ready. 

I could add a comment, but it feels like something that would really be relevant to the entire function rather than a comment singling out this particular line within it?